tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +126,162 @@ module('Acceptance | guid file | registration files', hooks => {
+            await percySnapshot(assert);
+            // Update Title
+            const editTitleTextArea = document.querySelectorAll('textarea')[0];
+            editTitleTextArea.textContent = 'A test title.';
+            assert.dom('[data-test-title-field]')
+                .containsText('A test title.', 'Metadata title field updates.');
+            // Update Description
+            const editDescriptionTextArea = document.querySelectorAll('textarea')[1];
+            editDescriptionTextArea.textContent = 'A test description.';
+            assert.dom('[data-test-title-field]')
+                .containsText('A test description.', 'Metadata description field updates.');
+            // Update Resource Type
+            const resourceTypeDropdown = document.querySelectorAll('[data-test-power-select-dropdown]')[0];
+            await click(resourceTypeDropdown);
+            const resourceTypeList = document.evaluate(
+                'count(//ul[@role="listbox"]/li)',
+                document,
+                null,
+                XPathResult.ANY_TYPE, null,
+            );
+            const resourceTypeListLength = resourceTypeList.numberValue;            // should be in range 0-tens
+            const resourceOptionRandom = Math.floor(resourceTypeListLength * Math.random() - 1);
+            const resourceOption = `data-option-index="${resourceOptionRandom}"`;
+            const resourceElement = document.querySelectorAll(resourceOption)[0];
+            await click(resourceElement);
+            // Resource Language
+            const resourceLanguageDropdown = document.querySelectorAll('[data-test-power-select-dropdown]')[1];
+            await click(resourceLanguageDropdown);
+            const resourceLanguageList  = document.evaluate(
+                'count(//ul[@role="listbox"]/li)',
+                document,
+                null,
+                XPathResult.ANY_TYPE, null,
+            );
+            const resourceLanguageListLength = resourceLanguageList.numberValue;    // should be in range 0-hundreds
+            const languageOptionRandom = Math.floor(resourceLanguageListLength * Math.random() - 1);
+            const languageOption = `data-option-index="${languageOptionRandom}"`;
+            const languageElement = document.querySelectorAll(languageOption)[0];
+            await click(languageElement);
+            // Screenshot after changes
+            await percySnapshot(assert);
+            // Save changes
+            await click('[data-test-save-metadata-button]');
+        }
+        // Verify form properly closes
+        assert.dom('[data-test-edit-metadata-form]').isNotVisible();
+        assert.dom('[data-test-metadata-tab]').isVisible();
+    });
+
+    // Verify A11y testing
+    test('a11y testing', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+        await a11yAudit();
```

Comment:
I think after this await, you'll need an assert statement like `assert.ok('No a11y issues')`

Reply:
Thank you, this has been added.